tests/components/devolo_home_network/test_binary_sensor.py

```diff
@@ -54,13 +52,9 @@ async def test_update_attached_to_router(
 
     state = hass.states.get(state_key)
     assert state is not None
-    assert state.state == STATE_OFF
-    assert state.attributes[ATTR_FRIENDLY_NAME] == f"{entry.title} Connected to router"
-
-    assert (
-        entity_registry.async_get(state_key).entity_category
-        == EntityCategory.DIAGNOSTIC
-    )
+    assert state == snapshot
+    entity_entry = entity_registry.async_get(state.entity_id)
+    assert entity_entry == snapshot
```

Comment:
I think this can be simplified further:
```python
    assert hass.states.get(state_key) == snapshot
    assert entity_registry.async_get(state.entity_id) == snapshot
```

Reply:
True, I simplified that.

Reply:
You can simplify even further.
```python
    assert hass.states.get(state_key) == snapshot
    assert entity_registry.async_get(state_key) == snapshot
```

You can also apply this change to the other platforms.

Reply:
I guess, I need more coffee. I thought, I checked for your first suggestion in the other touched platforms, too, but obviously I didn't see them. Now both simplifications are applied to all touched platforms.